### DISCUSSION TITLE
Hediye için Evleri dolaşan arkadaşlar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#16] [Zeka Sorusu]
Ahmet, Betül, Cevdet, Deniz ve Ertuğrul çembersel bir eğri üzerinde sırasıyla 1'den 29'a kadar numaralandırılmış evlerden oluşan bir sitenin merkezinde (çembersel eğrinin merkezinde) bulunmaktadırlar. Her biri için hazırlanmış hediye paketlerinin bulunduğu evi bilmedikleri için aramaya karar veriyorlar ve karışıklık çıkmaması için şu yöntemi uygulama konusunda anlaşıyorlar.

- Ahmet 1. evden başlayıp sırasıyla
- Betül 2. evden başlayıp birer ev atlayarak
- Cevdet 3. evden ikişer ev atlayarak
- Deniz 4. evden başlayıp üçer ev atlayarak
- Ertuğrul 5. evden başlayıp dörder ev atlayarak artan sırada numaralı evleri kontrol etmeye başlayıp aynı yönde devam edeceklerdir.

Her kontrolden sonra merkeze geri dönülmekte ve merkeze dönüş süresi her biri için aynıdır. Aramaya aynı anda başladıklarına ve hediyeler 13 numaralı evde bulunduğuna göre;

Soru 1: Eve 3. sırada kim ulaşır?

Soru 2: Eve 2. sırada ulaşanla 4. sırada ulaşan merkezde kaç kere karşılaşır?

Soru 3: Merkezde 2 kişi kaldığında, kalanlardan her biri kaç ev kontrol etmiştir?

**Comment by slos8888** [comment]
s1:Ahmet, Betül, Cevdet ve Deniz 13 numaralı eve ulaşamadan önce kontrolleri sona eriyor. Ancak, Ertuğrul 13 numaralı eve ulaşabiliyor. Bu nedenle, eve 3. sırada Ertuğrul ulaşır.
s2:Betül, 2. sırada, ve Deniz, 4. sırada eve ulaşırlar. Bu nedenle, merkezde birbirleriyle 2 kere karşılaşırlar.
s3: Merkezde en son 2 kişi kaldığında, kontrol ettikleri evler:

Betül: 2, 5, 8, 11, 14, 17, 20, 23, 26, 29 (10 ev)
Deniz: 4, 8, 12, 16, 20, 24 (6 ev)
Merkezde kalanlar, Betül 10 ev kontrol etmişti, Deniz ise 6 ev kontrol etmişti.
